### DISCUSSION TITLE
DETR 실험 결과

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#24] [Show and tell]
- train set loss 8(모든 head와 prediction의 로스합)
- lr 5e-6, epoch 80, pretrained
    - NMS의 문제에서 어느정도 벗어난 듯 하나, 배경과 물체가 비슷하면 문제가 생긴다(1행 4열 이미지)
    - 제출 성능 0.374(성능에 큰 향상 없음)
    - thr>0.3의 prediction result
        - 특이하게 큰 물체를 맞추는 능력은 떨어지나 작은 물체를 맞추는 능력은 탁월했다
        - 앙상블 가능성을 생각할 수 있을 듯 하다
        
thr = 0:
![image](https://user-images.githubusercontent.com/49234207/135593341-a9df6928-6f27-4707-bb33-accbea55101e.png)

thr = 0.3
![image](https://user-images.githubusercontent.com/49234207/135575846-5bbc7aca-74cc-4996-ac21-157cb15caa27.png)

thr = 0.5
![image](https://user-images.githubusercontent.com/49234207/135593475-31825c3b-4ed1-40c7-b9d8-343ccb2e2738.png)

thr = 0.7
![image](https://user-images.githubusercontent.com/49234207/135593502-fccfaf99-587d-43b8-94c3-23733a7bb301.png)

thr = 0.9
![image](https://user-images.githubusercontent.com/49234207/135593524-f5a91231-5461-4ebb-9ef6-0dd6e11b4a76.png)

**Comment by ppskj178** [comment]
라벨 추가했습니다.

**Comment by jaehwan-AI** [comment]
실험 결과 공유 감사합니다!!
